build.sh

```diff
@@ -114,6 +114,15 @@ if [ ! -d ${SYS_JDK7_HOME} ]; then
     exit 1
 fi
 
+build_warmup_stats() {
+    echo "\n===> Download and build stats\n"
+    if ! [ -d "${HERE}/warmup_stats" ]; then
+        cd ${HERE} && git clone https://github.com/softdevteam/warmup_stats || exit $?
+    fi
+
+    cd ${HERE}/warmup_stats && ./build.sh || exit $?
```

Comment:
This will use the system C compiler. You want to set it to zgcc like elsewhere.

I can take care of this. OK for me to add commits?

Reply:
I'll fix this one, but then after that, please go ahead!

Reply:
Actually, do we really care about this? After all, we're not *benchmarking* the warmup_stats repo...

Reply:
Hah, you are right.

Reply:
I have the fix locally, but I will roll it back.

I have some other small fixes I'd like to push if that's ok with you?

Reply:
OK, then consider that I have committed the null commit, thus fixing the problem ;) Go ahead with your commits.